### PR TITLE
两篇翻译

### PR DESCRIPTION
**Pull request by lujun9972**

### CODE REVIEW

File: Python Common/如何创建记录异常日志的装饰器.org [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#+TITLE: 如何创建记录异常日志的装饰器
+#+URL: http://www.blog.pythonlibrary.org/2016/06/09/python-how-to-create-an-exception-logging-decorator/
+#+AUTHOR: lujun9972
+#+CATEGORY: Python Common
+#+DATE: [2016-06-10 五 15:41]
+#+OPTIONS: ^:{}
+
+有一天,我突然想要创建一个用于捕获并记录一场的装饰器. 我在[[https://gist.github.com/diosmosis/1148066][Github]] 上发现一个颇为复杂的例子,该例子启发了我,并让我写下了以下代码:
```
Comment (ictar) [inline Python Common/如何创建记录异常日志的装饰器.org]: 一场 -> 异常

File: Python Common/如何创建记录异常日志的装饰器.org [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+              # log the exception
+              err = "There was an exception in  "
+              err += function.__name__
+              logger.exception(err)
+          # re-raise the exception
+          raise
+      return wrapper
+#+END_SRC
+
+在这段代码中,定义了两个函数. 地一个函数创建并返回了一个logging对象. 第二个函数为我们的装饰器函数,其中我们将传入的函数包裹进一个try/except中,并使用logger记录下任何产生的异常. 同时你还会发现,我还记录下了产生异常的函数名.
```
Comment (ictar) [inline Python Common/如何创建记录异常日志的装饰器.org]: 地一个 -> 第一个

File: Python Common/如何创建记录异常日志的装饰器.org [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   
+  @exception
+  def zero_divide():
+      1 / 0
+   
+  if __name__ == '__main__':
+      zero_divide()
+#+END_SRC
+
+当你在命令行中运行该代码后,你会发现多除了一个日志文件,其内容为:
```
Comment (ictar) [inline Python Common/如何创建记录异常日志的装饰器.org]: 多除 -> 多出

File: Python Common/如何创建记录异常日志的装饰器.org [hunk 4]
```diff
@@ -0,0 +1,72 @@
+#+TITLE: 如何创建记录异常日志的装饰器
```
Comment (ictar) [inline Python Common/如何创建记录异常日志的装饰器.org]: 对比了下原文，似乎没翻完。请保持文章完整再提交，谢谢~

File: Python Common/随机性的多种用途.org [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+[[https://en.wikipedia.org/wiki/Random_number_generation][计算机随机数生成器 ]](RNGs)[fn:1] 有很多用途. 最近在教授一次程序课时,我演示了随机数的一种用法,本来我觉得这是很普遍的一种用法,结果居然有一位参与者对这种用法表示很惊奇. 这件事情使得我意识到,对于许多初学者,甚至有些富有经验的程序员来说,恐怕对于随机数的很多用法都不是很熟悉.
+
+因此我想到了写这篇博客, 这里我会演示随机数的一些用法. 这里的例子使用的是python语言,但是其原理和概念也可以被借用于其他编程语言--只要它也能产生随机数.
+
+注意: 所有关于随机数的使用方法,都无需用到什么高深的数学知识.
+
+让我们开始吧.
+
+首先,从Python标准哭中导入并查看一下随机模块包含了那些内容
```
Comment (ictar) [inline Python Common/随机性的多种用途.org]: 哭？库？！
那些？哪些？！

**Comment by ictar** [comment]
几个误打的字改一下吧。然后把第一篇写完整，O(∩_∩)O谢谢~

**Comment by lujun9972** [comment]
我翻译的时候，当时还没有UPDATE
现在我又补全了UPDATE部分，也改了下错别字

**Comment by lujun9972** [comment]
Hi，我在翻译一篇文章时，有几句话不知道怎么翻译，求大神指点一下
https://github.com/lujun9972/emacs-document/blob/master/raw/Vim-Versus-Emacs-Minus-The-Religion.org中
１。　and there is a nearly infinite potential growth curve for how its various commands and shortcuts can be efficiently brought to bear in navigating and moving text around
２。　There’s nothing quite like having your editor dump core in the middle of a critical editing session to make you reconsider your choice of tools
３。　but chances are if you’re like most people you will get sucked in eventually as the lure of sanding the rough edges off of some mode or other becomes too great

**Comment by ictar** [comment]
１。　there is a nearly infinite potential growth curve for how its various commands and shortcuts can be efficiently brought to bear in navigating and moving text around
并且，对于它的各种命令和快捷方式如何能够有效地实现导航和文本移动，有近乎无限的潜在增长曲线

２。　There’s nothing quite like having your editor dump core in the middle of a critical editing session to make you reconsider your choice of tools
没有什么比你所使用的编辑器在你正进行一个关键的编辑会话时内存泄漏更能让你重新考虑对工具的选择了。

３。　but chances are if you’re like most people you will get sucked in eventually as the lure of sanding the rough edges off of some mode or other becomes too great
但如果你和大多数人一样，那么当把某些模式或者其他什么精雕细琢的诱惑变得太大时，最终很有可能你会陷进去而无法自拔。

—— 我尽力了，毕竟不是专业翻译~~

**Comment by lujun9972** [comment]
十分感谢　：）

**Pull request merged**